python/geometryApproximation.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+from paraview.simple import *
+
+# create a new 'XML PolyData Reader'
+streamlinesvtp = XMLPolyDataReader(FileName=['/home/emma/ttk/ttk-source/ttk-data-emmwel/GroundWater.cdb/streamlines.vtp'])
+
+# create a new 'XML PolyData Reader'
+stonevtp = XMLPolyDataReader(FileName=['/home/emma/ttk/ttk-source/ttk-data-emmwel/GroundWater.cdb/stone.vtp'])

```

Comment:
thanks for changing from an absolute to a relative path!
(i.e. just `GroundWater.cdb/stone.vtp`)